src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeCatalogController.php

```diff
@@ -26,6 +26,7 @@
 
 namespace PrestaShopBundle\Controller\Admin\Improve\Design;
 
+use PrestaShop\PrestaShop\Core\Foundation\Version\Version;
```

Comment:
Useless line

Reply:
@PierreRambaud I will fix it this evening with a new PR, thank you!